components/terraform/vault-provisioning/manifests/vaultpkisecret.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: secrets.hashicorp.com/v1alpha1
+kind: VaultPKISecret
+metadata:
+  name: vaultpkisecret-tls
+  namespace: tenant-1
+# https://developer.hashicorp.com/vault/docs/platform/k8s/vso/api-reference#vaultpkisecretspec
+spec:
+  # If no value is specified the Operator will default to the default VaultAuth,
+  # configured in its own Kubernetes namespace.
+  vaultAuthRef: default
+  # Namespace in Vault to get secrets from
```

Comment:
I think this is wrong, but the spec description is still very confusing (beta): https://developer.hashicorp.com/vault/docs/platform/k8s/vso/api-reference#vaultpkisecretspec

Reply:
fixed and confirmed in https://github.com/adfinis/Vault-Playground-on-Kubernetes/pull/1/commits/e7cd6a96a804645e8feb834bfa831a7314c74bde this is the k8s namespace for the authentication configuration, has nothing to do with the Vault namespace